src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,31 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param array $default Default limit for option selecting. Default is Paginator limit.
+     *
+     * @return string html output.
+     */
+    public function limitSelect($limits = [], $default = 0)
+    {
+        $out = $this->Form->create('', ['type' => 'get']);
```

Comment:
String is not a valid value for 1st argument for `FormHelper::create()` use `null` instead.